underscore.js

```diff
@@ -862,10 +862,10 @@
     var debounced = restArgs(function(args) {
       var callNow = immediate && !timeout;
```

Comment:
This is now only used inside the `if(immediate) {` block. Could we move it into that block?